external/anomaly/tests/conftest.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+try:
+    import e2e.fixtures
+
+    from e2e.conftest_utils import * # noqa
+    from e2e.conftest_utils import pytest_addoption as _e2e_pytest_addoption # noqa
+    from e2e import config # noqa
+    from e2e.utils import get_plugins_from_packages
+    pytest_plugins = get_plugins_from_packages([e2e])
+except ImportError:
+    _e2e_pytest_addoption = None
+    pass
```

Comment:
Please, remove this -- it is an obsolete version of the insertion

Note that `pytest_plugins = get_pytest_plugins_from_ote()` is added below -- it makes the newer version of the same changes.

Reply:
removed